servers/rendering/storage/render_scene_data.h

```diff
@@ -42,13 +42,22 @@ class RenderSceneData : public Object {
 protected:
 	static void _bind_methods();
 
+#ifndef DISABLE_DEPRECATED
+
+	Projection _get_cam_projection_93630();
+	Projection _get_view_projection_93630(uint32_t p_view);
+
+	static void _bind_compatibility_methods();
+
```

Comment:
```suggestion

	Projection _get_cam_projection_bind_compat_93630() const;
	Projection _get_view_projection_bind_compat_93630(uint32_t p_view) const;
	static void _bind_compatibility_methods();
```

Reply:
Hmm, in other places we have a line separating these, but happy to remove it.

Reply:
No worries that's minor, but the naming would be good

Reply:
owh totally missed the `compat_`.. doh!